plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ui/navigator/ConfigurationFileUtils.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.appengine.facets.ui.navigator;
+
+import static org.junit.Assert.fail;
+
+import com.google.cloud.tools.eclipse.appengine.facets.WebProjectUtil;
+import com.google.common.base.Strings;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.Path;
+
+/** Utility methods for creating App Engine configuration files for testing purposes. */
+public class ConfigurationFileUtils {
+
+  /** WebProjectUtil#createFileInWebInf() does not overwrite files. */
+  public static IFile createInWebInf(IProject project, IPath path, String contents) {
+    try {
+      IFile previous = WebProjectUtil.findInWebInf(project, path);
+      if (previous != null && previous.exists()) {
+        previous.delete(true, null);
+      }
+      return WebProjectUtil.createFileInWebInf(
+          project, path, new ByteArrayInputStream(contents.getBytes(StandardCharsets.UTF_8)), null);
+    } catch (CoreException ex) {
+      fail(ex.toString());
+      /*NOTREACHED*/
+      return null;
+    }
+  }
+
+  public static IFile createEmptyCronXml(IProject project) {
```

Comment:
If these aren't just for testing, it wouldn't hurt to have XML declarations and line breaks.